geniza/corpus/metadata_export.py

```diff
@@ -0,0 +1,223 @@
+import csv
+import os
+
+from django.conf import settings
+from django.contrib.postgres.aggregates import ArrayAgg
+from django.contrib.sites.models import Site
+from django.db.models.query import Prefetch
+from django.http import StreamingHttpResponse
+from django.utils import timezone
+from rich.progress import track
+
+script_user = settings.SCRIPT_USERNAME
+site_domain = Site.objects.get_current().domain.rstrip("/")
```

Comment:
This requires hitting the database, so you shouldn't set it as a global. Probably best to get it in exporter init

Reply:
In fact, this is probably why your unit tests are failing!